templates/unifi-poller.xml

```diff
@@ -3,25 +3,23 @@
   <Name>UniFi-Poller</Name>
   <Repository>golift/unifi-poller</Repository>
   <Registry>https://hub.docker.com/r/golift/unifi-poller/</Registry>
+  <Shell>sh</Shell>
   <Network>bridge</Network>
   <Privileged>false</Privileged>
-  <Support>https://hub.docker.com/r/golift/unifi-poller/</Support>
-  <Project>https://github.com/davidnewhall/unifi-poller</Project>
-  <Overview>Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus. Visualize with Grafana using included dashboards!&#xD;
-&#xD;
-This template works with poller versions 1.6.2+&#xD;
-&#xD;
-READ THE INSTRUCTIONS! &#xD;
-https://github.com/davidnewhall/unifi-poller/wiki/Installation</Overview>
-  <Category>Tools:Utilities Network:Management </Category>
+  <Changes>See https://github.com/unifi-poller/unifi-poller/wiki/CHANGES</Changes>
+  <Support>https://community.ui.com/questions/UniFi-Poller-Store-UniFi-Controller-Metrics-in-Prometheus-or-InfluxDB/58a0ea34-d2b3-41cd-93bb-d95d3896d1a1</Support>
```

Comment:
Did you read my "Do not do this"? :stuck_out_tongue:

Reply:
I did not lol